data-package/main.star

```diff
@@ -4,7 +4,7 @@ TAR_FILENAME = "dvd-rental-data.tar"
 # This Postgres data can be imported using `pg_restore` to populate a Postgres database
 def run(plan, args = {}):
     # From https://www.postgresqltutorial.com/postgresql-getting-started/postgresql-sample-database/
-    dvd_rental_data = plan.upload_files("./" + TAR_FILENAME)
+    dvd_rental_data = plan.upload_files("github.com/kurtosis-tech/awesome-kurtosis/data-package/" + TAR_FILENAME)
```

Comment:
hmm, is this not executed in CI on awesome-kurtosis????

Reply:
I think that there is something wrong on awesome-kurtosis CI, I don't see results for the executions:
https://app.circleci.com/pipelines/github/kurtosis-tech/awesome-kurtosis/1114/workflows/e857c957-d648-4930-8663-5182307c3579/jobs/3375